docs/modules/ROOT/pages/reference/extensions/openapi-java.adoc

```diff
@@ -39,6 +39,26 @@ Or add the coordinates to your existing project:
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
 
+== Usage
```

Comment:
You can move the usage content into `usage.adoc` and there's some [smart logic](https://camel.apache.org/camel-quarkus/2.7.x/contributor-guide/extension-documentation.html) in one of the maven plugins which will combine it into the extension docs.

Reply:
yeah, I create ```extensions/openapi-java/runtime/src/main/doc/usage.adoc```.